test/integration/echo_integration_test.cc

```diff
@@ -49,15 +49,16 @@ INSTANTIATE_TEST_SUITE_P(IpVersions, EchoIntegrationTest,
 TEST_P(EchoIntegrationTest, Hello) {
   Buffer::OwnedImpl buffer("hello");
   std::string response;
-  RawConnectionDriver connection(
+  std::unique_ptr<RawConnectionDriver> connection;
```

Comment:
Can you elaborate on why this change  was needed?

Reply:
I have literally no clue, it was before my time, and not suggesting this is the single best workaround.  Reverting the patch on msvc we end up with;
```
ERROR: C:/workspace/envoy/test/integration/BUILD:565:1: C++ compilation of rule '//test/integration:echo_integration_test_lib_internal_only' failed (Exit 2)
test/integration/echo_integration_test.cc(56): error C2065: 'connection': undeclared identifier
test/integration/echo_integration_test.cc(106): error C2065: 'connection': undeclared identifier
Target //test/integration:echo_integration_test failed to build
```
We are happy to withdraw changes to this specific file for separate consideration.

Reply:
We see an additional problem on Windows with this test. We will bypass it short term and are withdrawing the changes to echo_integration_test until they are better understood by all.